clingwrapper/src/clingwrapper.cxx

```diff
@@ -571,7 +571,32 @@ Cppyy::TCppScope_t Cppyy::GetScope(const std::string& name,
                                  + name + "'\n");
 #endif // NDEBUG
 
-    return Cpp::GetScope(name, parent_scope);
+    Cppyy::TCppScope_t scope = Cpp::GetScope(name, parent_scope);
+    if (!scope && (parent_scope == nullptr || parent_scope == Cpp::GetGlobalScope()))
+        scope = Cpp::GetScopeFromCompleteName(name);
```

Comment:
I was thinking about updating
```c++
TCppScope_t Cpp::GetScopeFromCompleteName(const std::string &name)
```
to 
```c++
TCppScope_t Cpp::GetScopeFromCompleteName(const std::string &name, TCppScope_t parent)
```
at CppInterOp.

Any suggestions?

Reply:
Yes, but I think the reason for this interface was to provide the fully scoped name. Do we benefit from providing the parent scope too?

Reply:
`Cpp::GetScope` does not handle namespaces, like `my_namespace::my_class`.
And `Cpp::GetScopeFromCompleteName` does not have the parent scope argument.

The code here will be simplified. Thats a benefit.